jac/jaclang/plugin/tests/fixtures/other_root_access.jac

```diff
@@ -67,15 +66,15 @@ walker allow_other_root_access {
         if self.via_all {
             Jac.unrestrict(here, self.level);
         } else {
-            Jac.allow_root(here, UUID(self.root_id), self.level);
+            Jac.allow_root(here, JacLangJID(self.root_id), self.level);
```

Comment:
i wonder if we can somehow not having separate handling for jid between jaclang and jac-cloud. It would be cool if the jac code is the same.

Reply:
for example, we define a plugin method that converts string to the corresponding JID type